cli/internal/client/client.go

```diff
@@ -17,9 +18,10 @@ import (
 
 type ApiClient struct {
 	// The api's base URL
-	baseUrl      string
-	Token        string
-	turboVersion string
+	baseUrl          string
+	Token            string
+	turboVersion     string
+	CurrentFailCount uint64
```

Comment:
Can you add a comment that this field is intended to be read and written from multiple goroutines? I know that's probably the default in Go, but it seems worth calling out because I don't think anything else in the struct is intended to be modified concurrently. See below for a note on `HttpClient` though.